src/Relations/BelongsToThrough.php

```diff
@@ -24,7 +32,7 @@ class BelongsToThrough extends Relation
     /**
      * The "through" parent model instances.
      *
-     * @var \Illuminate\Database\Eloquent\Model[]
+     * @var TIntermediateModel[]
```

Comment:
This is incorrect---`TIntermediateModel` is going to be a bound to a **single** model (like `User`), this is an array of different models not an array of `User`s

Reply:
Is there a way to let it know that it's the same list as the inputs? A generic which consists of a list of models?

Reply:
If it's updated to be an actual array then this could just become:

```suggestion
     * @var TIntermediateModels
```